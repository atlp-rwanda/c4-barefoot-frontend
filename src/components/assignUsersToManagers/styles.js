import { makeStyles } from '@material-ui/core/styles';

const styles = makeStyles((theme) => ({
    center: {
        textAlign: 'center'
    }, 


    Login: {
      height: '90vh',
    },
    paper: {
      margin: 'auto',
      display: 'flex',
      flexDirection: 'column',
      alignItems: 'center',
      justify: 'center',
      textAlign: 'center',
    },
    paper2: {
      margin: 'auto',
      display: 'flex',
      flexDirection: 'column',
      alignItems: 'center',
      justify: 'center',
      textAlign: 'center',
      color: 'white',
      height: '90%'
    },
    form: {
      width: '100%', 
      marginTop: theme.spacing(1),
    },
    login: {
      margin: theme.spacing(3, 0, 2),
      width: '40%'
    },
    submit: {
      margin: theme.spacing(3, 0, 12),
      width: '35%'
    },
    social_media: {
      margin: theme.spacing(3, 0, 2),
      width: '40%',
      height:'40px'
    },
    image_icon:{
        width: theme.spacing(3),
        height: theme.spacing(3)
    },
    social_media_sm: {
      margin: theme.spacing(3, 0, 2),
      width: '40%',
      height:'40px'
    },
    social_media_grid: {
      margin: 'auto',
      marginTop: '0',
      width: '80%',
      justifyContent: 'center',
      alignItems: 'center',
      textAlign: 'center'
    },
    secondpart: {
      margin: theme.spacing(5, 0, 5),
      width: '60%',
      justifyContent: 'center'
    },
    forgotPassword: {
      display: 'flex',
      justifyContent: 'flex-end',
    },
    
  }));

export default styles;
