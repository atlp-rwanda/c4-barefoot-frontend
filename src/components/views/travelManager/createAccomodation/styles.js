import { makeStyles } from "@material-ui/core";


export const useStyles= makeStyles( theme =>({
    container: {
        // backgroundColor: '#EAF4FB',
        width: '100vw',
        marginTop: '30px',
        marginBottom: '50px',
        display: 'flex',
        flexDirection: 'column',
        alignItems: 'center',
    },
    responsiveWrapper:{
        width: '100%',
        display: 'flex',
        justifyContent: 'center',
        padding:'0px 15px'
    },
    wrapperItem:{
        display:'flex',
        flexDirection: 'column',
        alignItems: 'center'
    },
    form: {
        width: '100%',
        marginTop: '40px',
    },
    formActions:{
        display: 'flex',
        justifyContent: 'center',
        marginTop:'50px'
    },
    formSection:{
        marginTop: '10px',
    },
    sectionHeader:{
        display: 'flex',
        justifyContent: 'space-between',
        padding: '20px 10px',
        backgroundColor: '#ABD5ED',
        transition: 'background-color .3s linear',
        '&:hover':{
            backgroundColor: '#43A0D6'
        }
    },
    sectionHeaderTitle:{
        display: 'flex'
    },
    headerIcon:{
        marginTop: '-2px',
        marginRight: '7px'
    },
    controlsSection:{
        width: '100%',
        // backgroundColor:'green',
        display: 'flex',
        flexDirection: 'column',
        alignItems: 'center',
    },
    groupTitle:{
        marginTop: '20px',
    },
    controlGroup: {
        width: '80%',
        // backgroundColor:"red",
        padding: '20px 0px',
        display: 'flex',
        justifyContent: 'center'

    },
    controlGroupAmenities: {
        width: '80%',
        // backgroundColor:"red",
        padding: '5px 0px',
        display: 'flex',
        justifyContent: 'center'

    },
    formControl:{
        width: '100%',
    },
    controlItem:{
        display: 'flex',
        justifyContent: 'center'
    },
    imageContainer:{

        backgroundColor: 'rgba(0,0,0,0.3)',
        minHeight: '280px',
        display: 'flex',
        flexDirection: 'column',
        alignItems: 'center',
    },
    imageIcon:{
        margin: 'auto',
        textAlign:'center'
    },
    button:{
        padding: '7px 20px',
        margin: '0px 10px',
        borderRadius: '0px'
    },
    resetBtn:{
        backgroundColor: '#F2994A',
        color: 'white'
    }



    


}));
