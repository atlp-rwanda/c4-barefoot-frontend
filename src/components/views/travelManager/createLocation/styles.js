import { makeStyles } from "@material-ui/core";


export const useStyles= makeStyles( theme =>({
    container: {
        width: '100vw',
        marginTop: '30px',
        marginBottom: '50px',
        display: 'flex',
        flexDirection: 'column',
        alignItems: 'center',
        // backgroundColor: 'gray',
    },
    responsiveWrapper:{
        width: '100%',
        display: 'flex',
        justifyContent: 'center',
        padding:'0px 15px'
    },
    wrapperItem:{
        display:'flex',
        flexDirection: 'column',
        alignItems: 'center',
        // backgroundColor:'#EAF4FB'
    },
    form: {
        width: '100%',
        marginTop: '40px',
        backgroundColor:'#EAF4FB',
        paddingBottom: '20px',

    },
    formActions:{
        display: 'flex',
        justifyContent: 'center',
        marginTop:'50px'
    },
    formSection:{
        marginTop: '10px',
    },
    
    controlsSection:{
        width: '100%',
        // backgroundColor:'green',
        display: 'flex',
        flexDirection: 'column',
        alignItems: 'center',
    },
    groupTitle:{
        marginTop: '20px',
        textAlign: 'center'
    },
    controlGroup: {
        width: '100%',
        padding: '20px 0px',
        display: 'flex',
        justifyContent: 'center'

    },
    formControl:{
        width: '100%',
    },
    controlItem:{
        display: 'flex',
        justifyContent: 'center'
    },
    imageContainer:{
        backgroundColor: 'rgba(0,0,0,0.3)',
        display: 'flex',
        flexDirection: 'column',
        alignItems: 'center',
    },
    respImageContainer:{
        display: 'flex',
        justifyContent: 'center'
    },
    imageIcon:{
        margin: 'auto',
        textAlign:'center'
    },
    button:{
        padding: '7px 20px',
        margin: '0px 10px',
        borderRadius: '0px'
    },
    resetBtn:{
        backgroundColor: '#F2994A',
        color: 'white'
    }  


}));
