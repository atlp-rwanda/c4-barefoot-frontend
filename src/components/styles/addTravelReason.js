import { makeStyles, fade } from '@material-ui/core/styles';

const useStyles = makeStyles((theme) => ({
    container:{
        justifyContent: 'center',
        alignItems: 'center'

    },
    insideGrid:{
        margin: theme.spacing(2,0,0,2)
    },
    title:{
        padding: theme.spacing(1),
        flexDirection:'column',
        textAlign: 'center',
        justifyContent: 'center',
        alignItems: 'center'
    },
    textFieldGrid:{
        justifyContent:'center',
        alignItems:'center',
        margin: theme.spacing(3,0),
        flexDirection: 'column'
    },
    buttons:{
        margin: theme.spacing(3),
        justifyContent: 'space-between'
    },
    cancelButton:{
        margin:theme.spacing(0,3)
    },
    textField:{
        width:'60%',
        [theme.breakpoints.down('sm')]:{
            width: '90%'
        }
    }
}))
 
export default useStyles;