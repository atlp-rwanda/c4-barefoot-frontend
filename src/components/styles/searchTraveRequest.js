import { makeStyles, fade } from '@material-ui/core/styles';
import colors from '../colors';
const useStyles = makeStyles((theme) => ({
    main:{
        backgroundColor: colors.primary200,
        width:'90%',
        margin:'auto',
        color:colors.neutralWhite,
        justifyContent: 'space-evenly',
        padding: theme.spacing(3),
    },
    searchLocation:{
        display:'flex',
        position: 'relative',
        width: '100%',
        backgroundColor: fade(theme.palette.common.white, 0.10),
        '&:hover':{
            backgroundColor: fade(theme.palette.common.white, 0.20)
        } ,
        padding: 0
    },
    searchIcon:{
        position:'absolute',
        padding: theme.spacing(0,1),
        height: '100%',
        display: 'flex',
        justifyContent: 'center',
        alignItems: 'center'
    },
    searchInput:{
        padding: theme.spacing(0,1,0,0),
        paddingLeft: '3em',
        color: 'inherit',
        width:'100%',
        border: '1px solid white'
    },
    dates:{
        padding: theme.spacing(1,0)
    },
    datesPicker:{
        color:colors.neutralWhite,
    },
    addButton:{
        color: colors.primary100,
        margin: theme.spacing(3,0,0,0)
    },
    searchCurrentLocation:{
        width: 300,
        padding:0,
        paddingLeft: '2em',

    },
    inputText:{
        color: colors.neutralWhite,
        margin: theme.spacing(0,2)
    },
    oneSelected:{
        margin:theme.spacing(0,0,1,0),
        color: colors.primary100,
        justifyContent:'flex-end',
        cursor: 'pointer',
    },
    closeSelectedLocations:{
        background:colors.red,
        display: 'block',
        color: colors.neutralWhite,
        borderRadius: '50%',
        padding: theme.spacing(0,1),
        paddingTop: '5px',
        paddingBottom:'5px',
        cursor: 'pointer',
        float:'right',
        fontWeight: 'bold',
        fontSize:'20px'
    },
    citiesSelected:{
        background:colors.neutralWhite,
        display:"block",
        padding: theme.spacing(1),
        borderRadius: '3px 15px 3px 3px',
        margin: theme.spacing(1),

    }
}))

export default useStyles;