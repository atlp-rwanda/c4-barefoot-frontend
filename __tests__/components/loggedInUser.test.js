import React from 'react';
import toJson from 'enzyme-to-json';
import LoggedInUserLayout from '../../src/components/layouts/LoggedInUser/LoggedInUser';
import { ThemeProvider } from '@material-ui/core/styles';
import { shallow } from 'enzyme';
describe('< LoggedInUser/>', () => {
    let wrapper;
    it("should be rendered without errors", () => {
        wrapper = shallow(
            <ThemeProvider theme={{ success: { main: '#fff' } }}>
                <LoggedInUserLayout />
            </ThemeProvider>
        );
        expect(toJson(wrapper)).toMatchSnapshot();
    })
});