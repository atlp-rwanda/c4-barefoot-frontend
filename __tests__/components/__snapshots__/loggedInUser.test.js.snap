// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`< LoggedInUser/> should be rendered without errors 1`] = `
<ContextProvider
  value={
    Object {
      "success": Object {
        "main": "#fff",
      },
      Symbol(mui.nested): false,
    }
  }
>
  <LoggedInUserLayout />
</ContextProvider>
`;
